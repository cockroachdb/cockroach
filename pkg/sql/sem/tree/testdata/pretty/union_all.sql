select a,b,c from x union all select d,e,f from y
