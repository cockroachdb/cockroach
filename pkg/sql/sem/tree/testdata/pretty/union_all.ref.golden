// Code generated by TestPretty. DO NOT EDIT.
// GENERATED FILE DO NOT EDIT
1:
-
SELECT
	a,
	b,
	c
FROM
	x
UNION ALL
	SELECT
		d,
		e,
		f
	FROM
		y

11:
-----------
SELECT
	a, b, c
FROM
	x
UNION ALL
	SELECT
		d,
		e,
		f
	FROM
		y

15:
---------------
SELECT
	a, b, c
FROM
	x
UNION ALL
	SELECT
		d, e, f
	FROM
		y

21:
---------------------
SELECT a, b, c FROM x
UNION ALL
	SELECT
		d, e, f
	FROM
		y

25:
-------------------------
SELECT a, b, c FROM x
UNION ALL
	SELECT d, e, f FROM y

31:
-------------------------------
SELECT a, b, c FROM x
UNION ALL SELECT d, e, f FROM y

53:
-----------------------------------------------------
SELECT a, b, c FROM x UNION ALL SELECT d, e, f FROM y


